src/main/java/org/opensearch/geospatial/index/common/xyshape/XYShapeConverter.java

```diff
@@ -28,6 +31,7 @@
 /**
  * Utility class to convert compatible shapes from opensearch to Lucene
  */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
why we're adding private constructor here comparing to URLBuilder class, is this for the sake of Lombok testing?

Reply:
got it, I just want to be sure we're minimizing additional changes and not mixing lombok support with other changes like refactoring etc.